src/main/java/pl/gacik/tictac/SignHolder.java

```diff
@@ -0,0 +1,22 @@
+package pl.gacik.tictac;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class SignHolder {
+
+    private Map<Sign, Player> signPlayerMap = new HashMap<>();
+
+    public void attachPlayer(Player player, Sign sign) throws IllegalArgumentException {
+        if(signPlayerMap.containsKey(sign)) {
+            throw new IllegalArgumentException("This sign has already booked by another player");
```

Comment:
"This sign has been already booked by another player" maybe?
or "This sign is booked by another player" ?